### PR TITLE
theme through checkout

### LINKED ISSUE #18: implement theme in checkout process

### PR DESCRIPTION
**Pull request by thogg4**
close #18

### CODE REVIEW

**Comment by thogg4** [comment]
@ericsaupe can you check this out?

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -13,4 +13,23 @@ def link_to_with_ti_icon(icon_name, text, url, options = {})
     link_to(text.html_safe, url, options)
   end
 
+  def link_to_for_cart(icon_name, text, url, options = {})
```
Comment (ericsaupe) [inline app/helpers/application_helper.rb]: This name seems confusing to me. So it is `link_to` but for the cart page or cart icon?

**Comment by ericsaupe** [comment]
You always beat me to issues! I was going to work on this tonight! Good job though, :panda_face:

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (thogg4) [inline app/helpers/application_helper.rb]: It should only be used in one place, so I didn't worry too much about the name.

Comment (ericsaupe) [inline app/helpers/application_helper.rb]: No problem, more just a question of what it was since the name confused me a bit.

**Pull request merged**